packages/core/src/Stores/client-store.js

```diff
@@ -398,6 +399,23 @@ export default class ClientStore extends BaseStore {
         });
     }
 
+    @action.bound
+    getLoginHistory(limit) {
+        return new Promise(resolve => {
+            WS.authorized.loginHistory(limit).then(data => {
+                runInAction(() => {
+                    if (data.error) {
+                        this.login_history = { api_initial_load_error: data.error.message };
```

Comment:
I think the component will be stuck in loading if we don't add `is_loading: false` here

Reply:
I'm going to remove the state from store so this will be fixed.